### ISSUE TITLE
Excuse me, when using the algorithm heavyhash and NVIDIA GPU to mine, it will be disconnected from the mining pool from time to time. The kernel is still showing the computing power, but the mining machine has been disconnected from the mining pool.

### ISSUE THREAD

**Issue by cheop1088** [andru-kun/wildrig-multi#155]
[Excuse me, when using the algorithm heavyhash and NVIDIA GPU to mine, it will be disconnected from the mining pool from time to time. The kernel is still showing the computing power, but the mining machine has been disconnected from the mining pool.
![图片](https://user-images.githubusercontent.com/113350461/189639160-f09e4a04-6fda-465e-8703-4bd9d274efc1.png)

**Comment by cheop1088** [comment]
When mining OBTC, it's always like this. When the kernel is opened, it can receive it effectively and quickly, then it gets slower and slower, and then when it stops there, it can display computing power normally.

**Comment by cheop1088** [comment]
![图片](https://user-images.githubusercontent.com/113350461/189640191-c8492546-b1b7-405d-a8f9-db0b066e7824.png)
The algorithm is suddenly much higher, then the information in the first picture appears, and then it is connected with the mine pool.

**Comment by andru-kun** [comment]
> ![图片](https://user-images.githubusercontent.com/113350461/189640191-c8492546-b1b7-405d-a8f9-db0b066e7824.png) The algorithm is suddenly much higher, then the information in the first picture appears, and then it is connected with the mine pool.

devfee just mining different algorithm, so it shows higher hashrate

**Comment by andru-kun** [comment]
> When mining OBTC, it's always like this. When the kernel is opened, it can receive it effectively and quickly, then it gets slower and slower, and then when it stops there, it can display computing power normally.

gpu just crashed, reduce your OC or use stock clocks

**Comment by cheop1088** [comment]
Hello, my GPU adjusted without overclocking.

**Comment by cheop1088** [comment]
Just adjusted the fan.

**Comment by cheop1088** [comment]
> > 挖OBTC的时候总是这样。  当内核打开时，它可以有效快速地接收它，然后越来越慢，然后停在那里，它可以正常显示计算能力。
> 
> gpu刚刚崩溃，减少你的OC或使用库存时钟

GPU does not overclock, only fans are used.

**Comment by andru-kun** [comment]
Unfortunately have no idea why it happens. Try maybe lower powerlimit for your gpu.

**Comment by cheop1088** [comment]
> 
Thank you very much for your reply. Can you help improve the software so that the 30 series high-end GPU card can run better

**Comment by andru-kun** [comment]
> Thank you very much for your reply. Can you help improve the software so that the 30 series high-end GPU card can run better

I have done everything I could about heavyhash algorithm in 2021 :)

**Comment by cheop1088** [comment]
> > 非常感谢您的回复。  能不能帮忙改进一下软件，让30系列高端GPU卡能跑得更好
> 
> 在 2021 年，我已经尽我所能来处理重哈希算法 :)

Ha ha, it's 2022, master

**Comment by andru-kun** [comment]
> Ha ha, it's 2022, master
Yeah, I know :) Mentioning 2021 I mean that I'm not working on heavyhash anymore, it's optimized already.
